shared/actions/chat/index.js

```diff
@@ -696,9 +696,8 @@ function * _selectConversation (action: Constants.SelectConversation): SagaGener
     yield put(Creators.clearMessages(conversationIDKey))
   }
 
-  let loadMoreTask
   if (conversationIDKey) {
-    loadMoreTask = yield fork(Saga.cancelWhen(_threadIsCleared, _loadMoreMessages), Creators.loadMoreMessages(conversationIDKey, true))
+    yield put(Creators.loadMoreMessages(conversationIDKey, true))
```

Comment:
Like above lets just dispatch the action. It doens't seem like we really need to join on this at all, seems to work fine. @MarcoPolo take a look at this since you wrote the original code